src/components/MiniBlog/index.tsx

```diff
@@ -43,7 +43,7 @@ const MiniBlog: React.FC = () => {
   const [jsonMdData, setJsonMdData] = useState<any>();
   const [value, setValue] = React.useState("all");
   const mobileBreakpoint = VIEW_PORT.MOBILE_BREAKPOINT;
-
+  const readingTime = require("reading-time");
```

Comment:
Not needed.

Reply:
sure, removed and updated using util function